src/Module/Cli/AdminAddUserCommand.php

```diff
@@ -55,11 +55,26 @@ public function execute(
         $values     = $this->values();
         $interactor = $this->io();
 
+        $email = $values['website'];
```

Comment:
is this a bug you found? i'd keep it simple with checking email and user separately
![image](https://user-images.githubusercontent.com/1305249/147995591-d2dd1557-c768-4979-a6b2-ba32a605f746.png)

Reply:
Yeah, I was able to add two users with the same email and username. I'm not going to touch migrations in here, but the email and username columns should probably be `UNIQUE`.